.travis.yml

```diff
@@ -0,0 +1,15 @@
+language: ruby
+rvm:
+  - 2.3.0
+env:
+  - DEFAULT_MAILER_HOST=localhost:1080 SECRET_KEY_BASE=asecretkeybase CAMPAIGN_ENDED=false MAILER_SENDER=mailer@example.com
```

Comment:
I'm seeing a different format for defining env vars in the docs for `.travis.yml`. Can you update this so it's one var per line?

Example here: https://docs.travis-ci.com/user/environment-variables/#Defining-public-variables-in-.travis.yml